Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -1296,30 +1231,31 @@ double EmbeddedFrag::mimicDistMatAndDensityCostFunc(
   } else {
     ddata = nullptr;
   }
-  unsigned int na = dp_mol->getNumAtoms();
-  unsigned int dsize = na * (na - 1) / 2;
+  auto na = dp_mol->getNumAtoms();
+  if (na < 2) {
+    return 0;
+  }
+  auto dsize = na * (na - 1) / 2;
   auto *ddata2D = new double[dsize];
```

Comment:
Switch to unique_ptr?

Reply:
it's added to a smart pointer on the next line